compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/ParserTestUtils.java

```diff
@@ -489,6 +489,8 @@ private static SyntaxKind getNodeKind(String kind) {
                 return SyntaxKind.BREAK_STATEMENT;
             case "RETURN_STATEMENT":
                 return SyntaxKind.RETURN_STATEMENT;
+            case "NIL_TYPE":
+                return SyntaxKind.NIL_TYPE;
```

Comment:
shall we move this to the 'Others' section in the switch case..?

Reply:
Sure @SupunS